api/account.md

```diff
@@ -1,78 +1,180 @@
-# Account control and info page for users and groups
-This page describes the API used for interacting with users and groups.
+# User Account APIs
 
-## Connectivity test
-#### allows for checking connectivity with the backend
-Basically a GET on /api/test returns a 200 with no content.  Pretty simple.
+This page describes the APIs for interacting with users.
 
-## Version API
-#### Path is /api/version/
-Perform a GET and you get version info.  Any, no authentication required.
+Requests sent to `/api/users` and other URLs under the `/api/users/` path operate on user accounts. The webserver will send StatusOK (200) on a good request, while a 400-500 status will be sent on error (depending on the error).
+
+The default "admin" account username cannot be changed or deleted, nor can the account be demoted out of admin status. Other user accounts may be also assigned administrator privileges.
+
+## Data types
+
+The APIs on this page primarily deal with user and group details structures. The user details structure is as follows:
+
+```
+{
+  UID: int,			// Numeric user ID
+  User: string,		// Username e.g. "jdoe"
+  Name: string,		// User's real name e.g. "John Doe"
+  Email: string,	// User's email address e.g. "john@example.org"
```

Comment:
Is the formatting here consistent? Looks like an uneven amount of whitespace.

Reply:
Regardless of what I do, it ends up rendered unevenly and terribly in the browser, even if I get it all nicely lined up in my editor. I eventually gave up.